lib/src/signed_video_h26x_sign.c

```diff
@@ -221,11 +220,11 @@ generate_sei_nalu(signed_video_t *self, uint8_t **payload, uint8_t **payload_sig
   SVI_TRY()
     // Get the total payload size of all TLVs. Then compute the total size of the SEI NALU to be
     // generated. Add extra space for potential emulation prevention bytes.
-    optional_document_size = tlv_list_encode_or_get_size(
-        self, optional_document_encoders, num_optional_document_encoders, NULL);
-    mandatory_document_size = !self->is_golden_sei
-        ? tlv_list_encode_or_get_size(self, mandatory_document_encoders, num_man_doc_encoders, NULL)
-        : 0;
+    optional_document_size =
```

Comment:
And since you're at it, maybe move away from "document" and use `optional_tags_size`
It is fine to rename `document_size` in this PR as well. Then use `tags_size`.